frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -101,19 +102,46 @@ class HtmlDocument extends React.Component {
         keepEditingWarning: true
       })
     }
+    // TODO test and review
+    let commentList = state.timeline.filter(item => item.timelineType === 'comment')
+    let {
+      actives_shares,
+      author,
+      created,
+      current_revision_id,
+      current_revision_type,
+      last_modifier,
+      ...revisionObject
+    } = data.content
```

Comment:
if I understand correctly, here, you are creating an object (`revisionObject`) by taking a subset of properties from `data.content`.
I understand the needs of this.
The only issue that I see with this is that, in the future, if properties are added to `data.content`, this code will have to be modified to remove the new properties (if they are not required). And I don't think there is any chance we think about this refactor.
Is it a problem that potentially added properties ends up in `revisionObject` ?
if it is, the solution would be to have a function to act as a serializer to create the `revisionObject` out of data.content

Reply:
TLM contains an object content.
The app, for the timeline, needs an object revision.
Here, it is the conversion.

question for backend: should we have a revision instead in the TLM ? or both ?